tools/basecurve/CMakeLists.txt

```diff
@@ -3,18 +3,18 @@ find_package(OpenMP)
 find_package(Exiv2 REQUIRED)
 include_directories(${EXIV2_INCLUDE_DIR})
 
-add_executable(darktable-curve-tool darktable-curve-tool.c exif-wrapper.cpp)
```

Comment:
this is wrong, please do not rename as the curve-tool can handle basecurve and tonecurve.